tests/pivots/spreadsheet_pivot/spreadsheet_pivot.test.ts

```diff
@@ -37,9 +37,9 @@ describe("Spreadsheet Pivot", () => {
       sheets: [
         {
           cells: {
-            A1: { content: "Customer" },
-            B1: { content: "Order" },
-            C1: { content: "Date" },
+            A1: "Customer",
+            B1: "Order",
+            C1: "Date",
```

Comment:
God there's a lot of these changes. We should probably go trough all of them sometime, I feel like 50% of the time we create a new model with hard-coded data, it's either 1) useless or 2) it would be easier to read AND less lines of code to use helpers `createModelFromGrid`, `resizeRows`, `setCellContent` ...